### PR TITLE
Task/adaptions 2

### PR DESCRIPTION
**Pull request by alonisser**

### CODE REVIEW

File: studies/migrations/0061_auto_20241026_1234.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    Stimulus = apps.get_model("studies", "Stimulus")
+    # get all categories that have any sub category
+    categories_with_sub_categories = StimulusCategory.objects.filter(stimulussubcategory__isnull=False).distinct()
+    # now filter out those that already have a sub category named "Other"
+    categories_without_other = categories_with_sub_categories.exclude(stimulussubcategory__name="Other")
+
+    for category in categories_without_other:
+        StimulusSubCategory.objects.create(name="Other", parent=category)
+
+    stimuli_to_process = Stimulus.objects.filter(category__in=categories_with_sub_categories).filter(
```
Comment (dani-efr) [inline studies/migrations/0061_auto_20241026_1234.py:18]: why would a category in `categories_with_sub_categories` have `sub_category__isnull=True`?

File: studies/resources/full_experiment.py [hunk 2]
```diff
@@ -124,22 +124,37 @@ def dehydrate_finding_tags(self, experiment: Experiment):
...
-                f"is_NCC: {finding.is_NCC}",
-                f"atlas_tag: {finding.AAL_atlas_tag}",
+        base_finding_resources = [
+            f"type: {finding.type.name}",
+            f"family: {finding.family.name}",
+            f"is_NCC: {finding.is_NCC}",
+            f"technique: {finding.technique}",
+            f"notes: {finding.notes}",
+        ]
+        if finding.family.name.lower() == "spatial" and finding.technique.name.lower == "fmri":
```
Comment (dani-efr) [inline studies/resources/full_experiment.py:134]: needs to be `finding.technique.name.lower()`

**Review by dani-efr — CHANGES_REQUESTED** [review]

File: studies/migrations/0061_auto_20241026_1234.py (hunk 1 above)
Comment (alonisser) [reply studies/migrations/0061_auto_20241026_1234.py:18]: it's not a "category" but a stimulus .
first we found general categories that "have" sub categories at all
then we checked if any of those don't have  a sub category of the type "Other"
then we get all stimulus that have those categories - but don't have a sub category
and for those we add the "Other" sub category

File: studies/resources/full_experiment.py (hunk 2 above)
Comment (alonisser) [reply studies/resources/full_experiment.py:134]: true, would fix

**Pull request merged**